server/routerlicious/packages/protocol-base/src/protocol.ts

```diff
@@ -67,10 +69,13 @@ export class ProtocolOpHandler {
     }
 
     public close() {
+        this.protocolState = undefined;
         this.quorum.close();
     }
 
     public processMessage(message: ISequencedDocumentMessage, local: boolean): IProcessMessageResult {
+        this.protocolState = undefined;
```

Comment:
I believe you want to move this line into default case, as otherwise there is not much use from new code - cache would be deleted on every op (even if op has nothing to do with protocol state)

Reply:
Every `processMessage` call changes the protocol state though because at least the sequence number will always change